src/main/scala/de/upb/cs/uc4/hyperledger/connections/traits/ConnectionOperationsTrait.scala

```diff
@@ -0,0 +1,55 @@
+package de.upb.cs.uc4.hyperledger.connections.traits
+
+import de.upb.cs.uc4.hyperledger.exceptions.traits.{ HyperledgerExceptionTrait, TransactionExceptionTrait }
+
+protected[hyperledger] trait ConnectionOperationsTrait extends ConnectionTrait {
+  final override val contractName: String = "UC4.Operation"
```

Comment:
```suggestion
  final override val contractName: String = "UC4.Approval"
```
We need to keep the old name until we support it in chaincode